source/glkernel/include/glkernel/sample.h

```diff
@@ -22,6 +22,9 @@ template <typename T, glm::precision P>
 size_t poisson_square(tkernel<glm::tvec2<T, P>> & kernel, T min_dist, unsigned int num_probes = 32);
 
 
+template <typename T, glm::precision P>
```

Comment:
please add comment in the following style:

``` c++
    /**
    *  @brief
    *    Bla ...
    *
    *  @param[in] name
    *    Bla ...
    *
    *  @return
    *    Bla ...
    */
```

in addition, comment on algorithm

* generate one jittered sample per row and column
* randomly shuffle the samples in x- or y-direction
* uniform distribution of the projections in x- and y-direction
* can generate an arbitrary number of stratified samples

(taken from http://cg.informatik.uni-freiburg.de/course_notes/graphics2_04_sampling.pdf)